addons/website/static/src/snippets/s_share/000.scss

```diff
@@ -7,8 +7,8 @@
     .s_share_title {
         margin: 0 .4rem 0 0;
     }
-    > a + a {
-        margin-left: .4rem;
+    a {
+        margin: .2rem;
```

Comment:
Because if the icons are on 2 lines we need a margin between lines.
Before: [https://tinyurl.com/v8djg7x](url)
After: [https://tinyurl.com/qqgjzps](url)